utils/autoReply.js

```diff
@@ -1 +1,22 @@
-// (for mentions) responds to a non-shamed user who mentions a shamed user with a message indicating the shamed status
+// If any user mentions a user who is focusing, they will get an auto-reply that lets the non-focused user know that the focused user is in focus mode and what time focus mode will end.
+const { msToString, remainingTime } = require('./parseTime');
+const { usersArray } = require('../commands/start');
+
+function autoReply(mentionMessage) {
+  //check all mentioned users to see if they are on list of focused users
+  const focusedUser = usersArray.map(
+    //work through array to see if user is in the list
+    // mentionMessage.mentions.users === whatever naming convention is
+  );
+
+  if(mentionMessage.mentions.users.has(focusedUser)) {
```

Comment:
use an `&&` here instead of nested ifs

`if(mentionMessage.mentions.users.has(focusedUser) && !mentionMessage.author.bot)`